src/app/core/components/spinner/spinner.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+import { LoadingService } from '@app/services/loading/loading.service';
+
+@Component({
+  selector: 'app-spinner',
+  templateUrl: './spinner.component.html',
+  styleUrls: [ './spinner.component.scss' ],
+})
+export class SpinnerComponent {
+  isLoading$: Observable<boolean>
```

Comment:
By the way, you can assign/link value just right here:
```
isLoading$: Observable<boolean> = this.loadingService.loading$;
```

Reply:
Yes, it can go as an option. But my approach seems to be a bit cleaner for me.